src/components/LoginForm/LoginForm.tsx

```diff
@@ -46,6 +46,7 @@ const LoginForm: React.FC<Props> = ({ onSubmit, onChange, values, errors, submit
         name="email"
         type="email"
         required
+        testId="email-input"
```

Comment:
Do we have other places in the app where maybe we have `email-input` used as a `testId`? Maybe here we could have something more specific like `login-email-input` 🙂

Reply:
Just kidding - April Fool's on me.  It is used in the login tests, but the string is interpolated so ctrl+F didn't find it. I added the login prefix as suggested and I also fixed the spelling (data-test-id instead of data-testid)